### PR TITLE
first commit

### PR DESCRIPTION
**Pull request by kuda1992**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/EUI-4243?filter=-1

### Change description ###

- Add a new route to get typesOfWork
- Add a new filter config 

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ x] No
```

### CODE REVIEW

File: src/work-allocation-2/components/task-list-filter/task-list-filter.component.ts [hunk 1]
```diff
@@ -1,9 +1,10 @@
-import { Component, EventEmitter, Input, OnDestroy, OnInit, Output } from '@angular/core';
+import { Component, EventEmitter, Input, OnDestroy, OnInit, Output, ViewEncapsulation } from '@angular/core';
 import { ActivatedRoute } from '@angular/router';
 import { FilterPersistence, FilterService } from '@hmcts/rpx-xui-common-lib';
 import { FilterConfig, FilterFieldConfig, FilterSetting } from '@hmcts/rpx-xui-common-lib/lib/models/filter.model';
-import { Subscription } from 'rxjs';
+import { forkJoin, Subscription } from 'rxjs';
 import { filter } from 'rxjs/operators';
+import { TaskTypesService } from 'src/work-allocation-2/services/task-types.service';
```
Comment (connorpgpmcelroy) [inline src/work-allocation-2/components/task-list-filter/task-list-filter.component.ts:7]: src

**Review by connorpgpmcelroy — CHANGES_REQUESTED** [review]
Just the src mention that is an issue

**Review by udaydenduluri33 — APPROVED** [review]
@kuda1992  thanks for the work type tasks feature. Thanks

**Review by connorpgpmcelroy — APPROVED** [review]
LGTM and comment addresses

**Pull request merged**